fastlane/docs/Actions.md

```diff
@@ -894,6 +894,20 @@ lane :refresh_dsyms do
   clean_build_artifacts           # Delete the local dSYM files
 end
 ```
+### [Sentry](https://getsentry.com)
+
+This action allows you to upload symbolication files to Sentry.
+
+```ruby
+sentry(
```

Comment:
Do you want this action to be called differently? Something like `upload_symbols_to_sentry`?